tests/headingcommand.js

```diff
@@ -171,13 +158,13 @@ describe( 'HeadingCommand', () => {
 				convertTo = option;
 			}
 
-			it( 'converts to default option when executed with already applied option', () => {
+			it( 'does nothing when executed with already applied option', () => {
 				const command = commands.heading1;
 
 				setData( document, '<heading1>foo[]bar</heading1>' );
 				command.execute();
 
-				expect( getData( document ) ).to.equal( '<paragraph>foo[]bar</paragraph>' );
+				expect( getData( document ) ).to.equal( '<heading1>foo[]bar</heading1>' );
```

Comment:
Hm... Good point about undo. This can be a good test. Apply heading 1 to heading 1 and see if that didn't create an undo step. Will you add a test for that, Olek?